devsetup/Makefile

```diff
@@ -176,11 +176,18 @@ standalone_snapshot: ## Create standalone snapshot
 	$(eval $(call vars))
 	virsh --connect=qemu:///system detach-device-alias edpm-compute-${EDPM_COMPUTE_SUFFIX} fs0 --live
 	virsh --connect=qemu:///system snapshot-create-as --atomic --domain edpm-compute-${EDPM_COMPUTE_SUFFIX} --name clean
+	if [ $? -ne 0 ] ; then
+		virsh --connect=qemu:///system suspend edpm-compute-${EDPM_COMPUTE_SUFFIX}
+		virsh --connect=qemu:///system destroy edpm-compute-${EDPM_COMPUTE_SUFFIX}
+		virsh --connect=qemu:///system snapshot-create-as --atomic --domain edpm-compute-${EDPM_COMPUTE_SUFFIX} --name clean
+	fi
 
 .PHONY: standalone_revert
 standalone_revert: ## Revert standalone snapshot
 	$(eval $(call vars))
 	virsh --connect=qemu:///system snapshot-revert --domain edpm-compute-${EDPM_COMPUTE_SUFFIX} --snapshotname clean
+	virsh --connect=qemu:///system resume --domain edpm-compute-${EDPM_COMPUTE_SUFFIX} ||:
+	$(warning Make sure the system time is synchronized for the reverted VM!)
```

Comment:
I don't get it this, especially after the double pipe, care to explain?

Reply:
that meant to ingore the resume command failures, as if we have creted a live snapshot, this command will fail to resume, but if we created the cold one, it needs to be resumed. Just a shortcut to suppress failures, but we can wrap it with the similar 'if else' section as above